vercel_runtime/src/http.rs

```diff
@@ -0,0 +1,96 @@
+use lambda_http::{http::StatusCode, Body, Error, Response};
+use serde::Serialize;
+use std::error::Error as StdError;
+use tracing::error;
+
+pub fn ok<T: Serialize>(val: T) -> Result<Response<Body>, Error> {
+    Ok(Response::builder()
+        .status(StatusCode::OK)
+        .header("content-type", "application/json")
+        .body(Body::Text(serde_json::to_string(&val).unwrap()))?)
+}
+
+#[derive(Serialize)]
+pub struct APIError {
+    pub message: &'static str,
+    pub code: &'static str,
+}
```

Comment:
I'm not sure we should be opinionated about error message/code formats. 

Might be nicer to just accept anything that is `Serialize` in the `bad_request` helper.

Reply:
Unsure on this one, because it's quite nice being consistent between the `404` / `400` / `500`.